geneLevel.R

```diff
@@ -376,13 +375,31 @@ if(!interactive()){
     parser$add_argument('-t', '--targetFile', type='character', default='IMPACT468', help="IMPACT341/410/468, or a Picard interval list file of gene target coordinates [default IMPACT468]")
     parser$add_argument('-m', '--method', type='character', default='reg', help="If scna, creates a portal-friendly scna output file [default reg]")
```

Comment:
I added `min_cf_cutoff` and `max_seg_length` as arguments via argparse as well. Feel free to remove, etc.

Reply:
Nice! Thanks. I'll update `max_seg_length` to be `10000000` for use with Allison's filters, and we'll leave the `25000000` hardcoded, since we want to eventually obsolete the TSG-based manual review.

Reply:
I see that @arichards2564 fixed `max_seg_length` to `10000000`. Marking as resolved.